apps/heft/src/plugins/TypeScriptPlugin/EmitFilesPatch.ts

```diff
@@ -24,6 +24,12 @@ export interface ICachedEmitModuleKind {
    */
   cacheOutFolderPath: string;
 
+  /**
+   * File extension to use instead of '.js' for emitted ECMAScript files.
+   * For example, '.cjs' to indicate commonjs content, or '.mjs' to indicate ECMAScript modules.
+   */
+  jsExtensionOverride: string | undefined;
```

Comment:
Do you need a corresponding change in `CopyStaticAssetsPlugin`?  Currently it duplicates assets into each [additionalModuleKindsToEmit ](https://github.com/microsoft/rushstack/blob/a529d2c0d0631b03bff7c05c6508e8cb46f76e20/apps/heft/src/plugins/CopyStaticAssetsPlugin.ts#L120) target folder.

Reply:
It uses a Set for the destination folders, so it's fine.